user.py

```diff
@@ -0,0 +1,14 @@
+class User:
+    email = 'no@no.com'
+    def __init__(name, age):
+        self.name = name
+        self.age = age
+
+    def user_print():
+        print("Name: "+ self.name + "; Age: "+ self.age)
+
+    def execute(cmd):
+        try:
+            retcode = subprocess.call(cmd, shell=True)
```

Comment:
Recommendation generated by Amazon CodeGuru Reviewer. Leave feedback on this recommendation by replying to the comment or by reacting to the comment using emoji.

It appears that you are using a string to construct a shell command that invokes the shell explicitly, via `shell=True`. This might be susceptible to shell injection. We recommend that you use a `shlex.quote()` function to properly escape whitespace and shell meta-characters in strings that are used to construct shell commands.

[Learn more](https://docs.python.org/3/library/subprocess.html#security-considerations)

Reply:
That is great 👍